website/docs/config-file.md

```diff
@@ -33,6 +33,7 @@ backtraces_path = "/path/to/neovide_backtraces.log" # see below for the default
 fork = false
 frame = "full"
 idle = true
+icon = "/full/path/to/neovide.ico" # for macOS the icon should be in `.icns` format.
```

Comment:
This is not a default value. The value should have the value that would be used by default give the line above the code block

Reply:
no this is about the docs not the functionality. maybe take another look?

Reply:
thank you, misunderstood. check again please.